src/components/pages/HomePage/index.jsx

```diff
@@ -36,7 +30,9 @@ import { ReactComponent as DeployImg } from "../../../assets/content/svg/deploy.
 import { ReactComponent as ShopImg } from "../../../assets/content/svg/shop.svg";
 import { ReactComponent as VaultImg } from "../../../assets/content/svg/vault.svg";
 import { ReactComponent as PartyImg } from "../../../assets/content/svg/party.svg";
+//#endregion
 
+//#region > Functions
 let backgroundPictures = [baristaBg, breadBg, shopBg];
```

Comment:
This can be a `const`.

```suggestion
const backgroundPictures = [baristaBg, breadBg, shopBg];
```